packages/financial-templates-lib/test/clients/OptimisticOracleEventClient.js

```diff
@@ -60,49 +59,66 @@ contract("OptimisticOracleEventClient.js", function (accounts) {
   const defaultAncillaryData = "0x";
 
   const pushPrice = async (price) => {
-    const [lastQuery] = (await mockOracle.getPendingQueries()).slice(-1);
-    await mockOracle.pushPrice(lastQuery.identifier, lastQuery.time, lastQuery.ancillaryData, price);
+    const [lastQuery] = (await mockOracle.methods.getPendingQueries().call()).slice(-1);
+    await mockOracle.methods
+      .pushPrice(lastQuery.identifier, lastQuery.time, lastQuery.ancillaryData, price)
+      .send({ from: accounts[0] });
   };
 
   before(async function () {
-    finder = await Finder.new();
-    timer = await Timer.new();
-
-    // Whitelist an initial identifier we can use to make default price requests.
-    identifierWhitelist = await IdentifierWhitelist.new();
-    await identifierWhitelist.addSupportedIdentifier(identifier);
-    await finder.changeImplementationAddress(utf8ToHex(interfaceName.IdentifierWhitelist), identifierWhitelist.address);
-
-    collateralWhitelist = await AddressWhitelist.new();
-    await finder.changeImplementationAddress(utf8ToHex(interfaceName.CollateralWhitelist), collateralWhitelist.address);
+    accounts = await web3.eth.getAccounts();
+    [owner, requester, proposer, disputer] = accounts;
 
-    store = await Store.new({ rawValue: "0" }, { rawValue: "0" }, timer.address);
-    await finder.changeImplementationAddress(utf8ToHex(interfaceName.Store), store.address);
+    finder = await Finder.new().send({ from: accounts[0] });
+    timer = await Timer.new().send({ from: accounts[0] });
 
-    mockOracle = await MockOracle.new(finder.address, timer.address);
-    await finder.changeImplementationAddress(utf8ToHex(interfaceName.Oracle), mockOracle.address);
+    // Whitelist an initial identifier we can use to make default price requests.
+    identifierWhitelist = await IdentifierWhitelist.new().send({ from: accounts[0] });
+    await identifierWhitelist.methods.addSupportedIdentifier(identifier).send({ from: accounts[0] });
```

Comment:
as a general comment, it's much cleaner IMHO to use the `owner` over the `account[0]`. I know you did not add this in this PR. We should consider going through and removing this syntax through the protocol tests when there is a variable declared that holds the same address.

Reply:
Agreed. I started doing it later since I was having to define the named accounts anyway (so there was no risk of owner not existing in that file). 

We can do a big find/replace down the line, but honestly, I just want to get the tests working, and I'm not _super_ concerned as long as it's easy to understand what's going on.